CONTRIBUTING.md

```diff
@@ -273,52 +288,90 @@ Non-exhaustive list of things that make it *easier* to review:
 Note that these lists are *independent* of how simple/complicated the actual
 *code* changes are.
 
-## Draft Pull Requests
-
-If you want early feedback on your PR, use GitHub's "Draft Pull Request"
-mechanism. Draft PRs are a convenient way to collaborate with the Agave
-maintainers without triggering notifications as you make changes. When you feel
-your PR is ready for a broader audience, you can transition your draft PR to a
-standard PR with the click of a button.
-
-Do not add reviewers to draft PRs.  GitHub doesn't automatically clear
-approvals when you click "Ready for Review", so a review that meant "I approve
-of the direction" suddenly has the appearance of "I approve of these changes."
-Instead, add a comment that mentions the usernames that you would like a review
-from. Ask explicitly what you would like feedback on.
-
 ## Rust coding conventions
 
 * All Rust code is formatted using the latest version of `rustfmt`. Once
   installed, it will be updated automatically when you update the compiler with
-`rustup`.
+  `rustup`. To run the linter manually, execute `cargo +nightly fmt`.
```

Comment:
They way I run `fmt` is like this:
```bash
./scripts/cargo-for-all-lock-files.sh -- +(bash -c 'source ci/rust-version.sh; echo $rust_nightly') fmt --all
```
Not sure if we want to encode it into our instructions or not.
It would be much better to have a more user-friendly command or script that does the same.  I've put some effort into it at some point, but did not really finish it.

Reply:
note that `./ci/test-checks` verifies there are no violations, but doesn't fix them... I find it mildly annoying, most of the time there are auto-fixable no-brainer violations during development, it would be best to have script that detects + fixes them

Reply:
Agreed! But we do not have one -> out of scope here?

Reply:
For the purpose of this PR I agree though that including a complicated command as recommended could be distracting when `test-checks` does the job of signaling violations

Reply:
nothing is ever perfect. We can make a relevant script in a followup.